### DISCUSSION TITLE
HmIP-HAP einbinden nur mit HmIP-RFUSB?

### DISCUSSION THREAD

**Discussion by phillg80** [OpenCCU/OpenCCU#2643] [🇩🇪 Fragen&Antworten] [answered]
Moin zusammen,

benötige ich für das Einbinden des HAP zwingend immer den RF USB? Und wird dieser Stick nach dem Einbinden immer noch benötigt?

**Comment by jens-maus** [comment, marked as answer]
> benötige ich für das Einbinden des HAP zwingend immer den RF USB? Und wird dieser Stick nach dem Einbinden immer noch benötigt?

Man braucht für die Nutzung eines `HmIP-HAP` sowie eines `HmIPW-DRAP` **zwingend** das `RPI-RF-MOD` oder `HmIP-RFUSB`. Und das nicht nur für das Einbinden, sondern auch für den dauerhaften Betrieb, denn bei diesen beiden handelt es sich nicht nur um "Funkmodule", sondern eben auch Koprozessoren die für die homematicIP Verschlüsselung notwendige Komponenten mitbringen.

All diese Informationen kann man übrigens der entsprechenden "Vorraussetzungen" Tabelle hier entnehmen:
https://github.com/jens-maus/RaspberryMatic/wiki/Einleitung#voraussetzungen

**Reply by phillg80** [reply]
Was für ein Support! Vielen Dank für die so schnelle Antwort. 

Weisst du, ob der Telekom Homematic IP Stick kompatibel bzw. sogar baugleich ist?

**Reply by jens-maus** [reply]
> Weisst du, ob der Telekom Homematic IP Stick kompatibel bzw. sogar baugleich ist?

Auch das kann man aus der verlinkten Tabelle und zahlreichen Foreneinträge problemlos ablesen! Nein, der `HmIP-RFUSB-TK` (die Telekom OEM Variante) ist **nicht** mit allem kompatibel. So kann er nur reines homematicIP und kein altes BidCos-RF und auch kann man ihn nicht zur Anbindung eines `HmIP-HAP` oder `HmIPW-DRAP` nutzen wie das mit dem originalen `HmIP-RFUSB` der Fall ist.
